### PR TITLE
Implement Mixpanel

### PR DESCRIPTION
**Pull request by jack-toribio-adsk**

### CODE REVIEW

File: src/mixpanel.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import mixpanel from 'mixpanel-browser';
+// mixpanel.init('52d9105dc2cbd94e7b0da50a2b65f4c0');
+mixpanel.init('52d9105dc2cbd94e7b0da50a2b65f4c0', { debug: true, track_pageview: true, persistence: 'localStorage' });
+
+
+let env_check = true; // process.env.NODE_ENV === 'production';
+
+let actions = {
```
Comment (pouzol-adsk) [inline src/mixpanel.tsx:8]: are those actions usefull ? or is it default ?

**Review by pouzol-adsk — COMMENTED** [review]
Depending on the comment answer, the rest looks good to me

File: src/mixpanel.tsx (hunk 1 above)
Comment (jack-toribio-adsk) [reply src/mixpanel.tsx:8]: track, identify, alias and people are common actions

**Review by pouzol-adsk — APPROVED** [review]
LGTM

**Pull request closed without merging**